Dockerfile

```diff
@@ -78,6 +79,11 @@ COPY engine /engine
 COPY should_run_cluster_collector.rb /should_run_cluster_collector.rb
 COPY --chmod=755 cluster-collector.sh /cluster-collector.sh
 COPY --chmod=755 ebpf.sh /ebpf.sh
+# Copy default enrichment files to both locations
+# /enrichment-defaults is the source for copying at runtime
+# /enrichment is for backwards compatibility when not using volumes
```

Comment:
For what scenario do we care about backwards compatibility here please? Older VRL with new collector?